assets/sass/05-blocks/separator/_editor.scss

```diff
@@ -42,3 +41,15 @@ hr {
 		border-color: currentColor;
 	}
 }
+
+.wp-block[data-align="full"],
```

Comment:
This needs to be inside the .wp-block-separator, otherwise the default separator will be full width when placed inside another full width block like a group.

Reply:
> This needs to be inside the .wp-block-separator, otherwise the default separator will be full width when placed inside another full width block like a group.

It wouldn't work this way since the element with the class `.wp-block[data-align=""]` always wrap the separator (when set as full or wide width) as parent.

What adding instead is adding the `>` selector before the separator class. Then this 
```
.wp-block[data-align="wide"] {

	> .wp-block-separator,
	> hr {

		&:not(.is-style-dots) {
			max-width: inherit;
		}
	}
}
```
will only applies if the separator is the direct children of an element with the class `.wp-block[data-align=""]`.

Reply:
I have been placing it inside the .wp-block-name class, and then used the `&` as a parent selector, but I did not know sass before this project started, so I don't know best practices.

https://sass-lang.com/documentation/style-rules/parent-selector

Reply:
I don't think there are any set standards around this, but we do use the `&` a lot to avoid repetition. I'd suggest moving it up into the `.wp-block-separator, hr` block, and in fact you don't need the `:not(.is-style-dots)` specificity hack, so it can be simplified to:

```scss
[data-align="full"] > &,
[data-align="wide"] > & {
	max-width: inherit;
}
```